shared/todo.test.js

```diff
@@ -13,56 +12,67 @@ beforeEach(() => {
   deleteFile(`${__dirname}/done.txt`);
 });
 
-let todoTxtCli = (...args) => [`${__dirname}/todo.sh`, ...args].join(" ");
+let executable = process.platform === "win32" ? "todo.bat" : "todo.sh";
+
+let todoTxtCli = (...args) => [`${__dirname}/${executable}`, ...args].join(" ");
 
 let usage = `Usage :-
-$ ./todo.sh add "todo item"  # Add a new todo
-$ ./todo.sh ls               # Show remaining todos
-$ ./todo.sh del NUMBER       # Delete a todo
-$ ./todo.sh done NUMBER      # Complete a todo
-$ ./todo.sh help             # Show usage
-$ ./todo.sh report           # Statistics
-`;
+$ ./todo.exe add "todo item"  # Add a new todo
+$ ./todo.exe ls               # Show remaining todos
+$ ./todo.exe del NUMBER       # Delete a todo
+$ ./todo.exe done NUMBER      # Complete a todo
+$ ./todo.exe help             # Show usage
+$ ./todo.exe report           # Statistics`;
 
```

Comment:
The reason I used `exe` was to simply _indicate_ that this is how the _executable_ is supposed be used. Since there is no unified executable (`sh` on *nix, and `bat` on Windows), I thought it made sense to avoid favoring one platform over another.

I've tested symbolic links, and it appears to work. We can remove the extension from the test's expectation for the help command, and add an instruction in the README to create a symbolic link? Something along the lines of:

> ### Create symbolic link to the executable file
>
> Before running tests, create a symbolic link to the executable used on your platform - `todo.bat` on Windows, and `todo.sh` on *nix systems:
>
> **Windows:**
> ```
> > mklink todo todo.bat
> ```
>
> ***nix:**
> ```
> $ ln -s todo.bat todo
> ```

Reply:
Sounds good. Can you add this section to the README.